### PR TITLE
add type hinting for plotly

### PR DESCRIPTION
**Pull request by adhami3310**
some changes are breaking

### CODE REVIEW

File: reflex/components/plotly/plotly.py [hunk 1]
```diff
@@ -116,49 +117,49 @@ class Plotly(NoSSRComponent):
...
-    on_autosize: EventHandler[_event_data_signature]
+    on_autosize: EventHandler[empty_event]
 
     # Fired whenever mouse moves over a plot.
-    on_before_hover: EventHandler[_event_data_signature]
+    on_before_hover: EventHandler[empty_event]
 
     # Fired when a plotly UI button is clicked.
-    on_button_clicked: EventHandler[_button_click_signature]
+    on_button_clicked: EventHandler[empty_event]
```
Comment (masenf) [inline reflex/components/plotly/plotly.py:138]: why is this one no longer passing through the button information

**Review by masenf — COMMENTED** [review]
why was `_event_data_signature` removed?

File: reflex/components/plotly/plotly.py (hunk 1 above)
Comment (adhami3310) [reply reflex/components/plotly/plotly.py:138]: i have tried the event and it always printed null

**Comment by adhami3310** [comment]
> _event_data_signature

because it's a really big event class that mostly transferred as an empty dict

**Review by masenf — APPROVED** [review]

**Pull request merged**